.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,55 @@
+name: "Auto release"
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
```

Comment:
I think this will not work, as this will just be triggered on every PR that is merged? 😅 this works in sentry-javascript because of gitflow, where we only merge into master when we actually release. here we'll need some other check I guess?

Reply:
Right, forgot to add the check to only run on `prepare-release` branches. Added this now.